src/framework/mlt_repository.c

```diff
@@ -101,6 +101,24 @@ mlt_repository mlt_repository_init( const char *directory )
 	mlt_tokeniser tokeniser = mlt_tokeniser_init();
 	int dl_length = mlt_tokeniser_parse_new( tokeniser, getenv( "MLT_REPOSITORY_DENY" ), ":" );
 
+	// check if both qt5 and qt6 modules are available…
+	int qtmodules = 0;
+	for ( i = 0; i < count; i++ )
+	{
+		const char *object_name = mlt_properties_get_value( dir, i);
+		qtmodules += strncmp(object_name, "mltqt", strlen( "mltqt" ) );
+		qtmodules += strncmp(object_name, "mltqt6", strlen( "mltqt6" ) );
+	}
+	// …and not blocked
+	for (int j = 0; j < dl_length; j++ )
+	{
+		char *denyfile = calloc( 1, strlen( directory ) + strlen( mlt_tokeniser_get_string( tokeniser, j ) ) + 2 );
```

Comment:
This needs to be `+ 3` for the trailing null character because `+ 2` only accounts for the `/` and `.` characters in the format string.

Reply:
I know why I usually develop in high level C++ :laughing: